### PR TITLE
1-implement-localization-feature-and-refactor-shitty-code

### PR DESCRIPTION
**Pull request by bladuk**

### CODE REVIEW

File: src/modules/localization.ts [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    AveragePlayers = 'AVERAGE_PLAYERS',
+    PeakPlayers = 'PEAK_PLAYERS',
+    InvalidCommunity = 'INVALID_COMMUNITY',
+    InvalidServer = 'INVALID_SERVER',
+    CacheNotUpdated = 'CACHE_NOT_UPDATED',
+    CommandError = 'COMMAND_ERROR'
+}
+
+let currentLocales: ILocale[] = [];
+let defaultLocale:ILocale;
```
Comment (Vladislav-Antipov) [inline src/modules/localization.ts:69]: space

**Review by Vladislav-Antipov — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
   "name": "scpboard",
   "version": "1.0.0",
   "description": "A discord bot which allows you to get information about SCP:SL servers",
-  "main": "./dist/index.js",
-  "exports": "./dist/index.js",
+  "main": "./src/index.ts",
```
[the remark this answers is not in the document]
Comment (Vladislav-Antipov) [reply package.json:5]: wait

File: src/commands/discord/contribute.ts [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
 export default {
     data: new SlashCommandBuilder()
         .setName('contribute')
         .setDescription('Displays information about how to contribute this bot'),
     async execute(interaction: ChatInputCommandInteraction) {
-        let answer = 'If you would like to contribute it, you can update or add your native language to this bot! Click [here]() to read README.';
-        answer += '\nCurrent contributors:\n';
-
-        // translators
+        let answer = readLocalizationKey(interaction.locale, TranslationKey.Contribute, "NO_TRANSLATION") + getTranslatorsMessage();
```
[the remark this answers is not in the document]
Comment (bladuk) [reply src/commands/discord/contribute.ts:9]: Already implemented, open your eyes

[the remark this answers is not in the document]
Comment (Vladislav-Antipov) [reply src/commands/discord/contribute.ts:9]: then delete 3th argument since it has default value

[the remark this answers is not in the document]
Comment (bladuk) [reply src/commands/discord/contribute.ts:9]: It's your part of code

**Review by Vladislav-Antipov — APPROVED** [review]

**Pull request merged**